duckduckgo/iOS
# alignment with site grade logic from extensions team

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/458611166160477/f
CC:

**Description**:

Aligns the Site Rating with the extensions team and tries to make main logic section look similar so it's easier to track future changes.

Additionally, fixes some feedback raised with initial screen (which are mostly related to this work):

* Privacy Grade from D to B doesn't show https://app.asana.com/0/413877547933097/312386120368398/f

* Show privacy grade while loading: https://app.asana.com/0/413877547933097/312386120368396/f

* Slow protection toggle: https://app.asana.com/0/413877547933097/312386120368391/f

* Unknown privacy practices should be rated B max: https://app.asana.com/0/413877547933097/312386120368386/f

* Minor change to improve UX of dismissing privacy protection while under consideration by Mike: https://app.asana.com/0/413877547933097/312386120368389/f

Also updated the assets with the vector PDFs.

Note: have added hooks to handle non-secure content as we currently base https purely on the site's URL, but that's too big a change for here, so will create a new Asana to integrate that.

**Steps to test this PR**:

Test 1:
1. Visit Evans Cycles and launch the Privacy Protection
1. Note the upgrade message
1. Turn off privacy protection
1. Note that the toggle is instant
1. Note the grade updates
1. Turn on privacy protection
1. Note the upgrade message

Test 2:
1. Visit news.ycombinator.com
1. Privacy Protection 
1. Note the B grade

Test 3:
1. Visit a site and launch the Privacy Protection
1. Tap the omni bar text field
1. The privacy protection is dismissed and the text field takes focus


###### Reviewer Checklist:


###### PR DRI Checklist:
